crates/rattler_lock/src/pypi.rs

```diff
@@ -59,6 +59,7 @@ impl Ord for PypiPackageData {
         self.name
             .cmp(&other.name)
             .then_with(|| self.version.cmp(&other.version))
```

Comment:
can you also add the hash as a last resort?
Or maybe the platform or something like that, if we have that?